windows/datetimepicker.cpp

```diff
@@ -189,3 +189,15 @@ uiDateTimePicker *uiNewTimePicker(void)
 {
 	return finishNewDateTimePicker(DTS_TIMEFORMAT);
 }
+
+void uiDateTimePickerTime(uiDateTimePicker *d, struct tm *time)
```

Comment:
For this, you will have to populate all the fields of `struct tm` manually from a `SYSTEMTIME` structure. I can take care of it if you'd prefer.

Reply:
I wanted to get your feedback before working on the Windows implementation. Will do it next after fixing the other issues you've brought up here.

Reply:
I wanted to get your feedback before working on the Windows implementation. Will do it next after fixing the other issues you've brought up here.

Reply:
Windows seems to need `tm_wday` set correctly for some reason.